packages/toolpad-app/src/runtime/ToolpadApp.tsx

```diff
@@ -1226,7 +1228,7 @@ function QueryNode({ page, node }: QueryNodeProps) {
     Object.fromEntries(node.params ?? []),
   );
 
-  const configBindings = _.pick(node.attributes, USE_DATA_QUERY_CONFIG_KEYS);
+  const configBindings = filterKeys(node.attributes, (key) => USE_DATA_QUERY_CONFIG_KEYS.has(key));
```

Comment:
We could abstract this to its our own `pick` function maybe?

Reply:
Removing it altogether